### PR TITLE
Identifications_chart

### PR DESCRIPTION
**Pull request by KNAthari**
### Changes
- Created four different boxes for identifications
- Scoped each box by a particular perspective
- Seeded test data 

### Link to Issue 
(https://app.asana.com/0/1206510424537042/1207173731090929/f)

### Comments, Notes, or  Questions
This section is if you want to share something related to the PR with the reviewer. 

### Related Screenshots 
For FE tickets specifcially, please include a screenshot of your work!
Screenshot 2024-05-30 at 11 00 04 AM
Screenshot 2024-05-30 at 10 59 53 AM

### Before you submit for review, Did you...

- [x] Added relevant tests (if applicable)
- [x] Linked Asana ticket to PR
- [x] Added comments, notes, or questions for reviewwer
- [x] Added relevant screenshots
- [x] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

File: app/views/identifications/index.html.erb [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
-          when 'hold_back' then 'bg-alert-dark'
-          when 'neutral' then 'bg-warning-dark'
-          when 'support' then 'bg-success-dark'
-          else 'bg-notice-light'
-        end %>
-        <!-- identification as a circle with the first letter (ACD) -->
-        <div class='w-24 h-24 flex justify-center items-center rounded-full <%= circle_color %> text-8xl font bold'>
-        <%= identification.way_of_being_name.first.upcase %>
+<div class='flex flex-wrap w-full h-full bg-gray-100 p-8'>
+  <% [@identifications_box1, @identifications_box2, @identifications_box3, @identifications_box4].each do |identifications|%>
```
Comment (edithemmings) [inline app/views/identifications/index.html.erb:2]: ^where are these variables defined?

Comment (KNAthari) [reply app/views/identifications/index.html.erb:2]: The instance variables are meant to create the four boxes. They are just placeholders, defines only after the loop iterates through them.

Comment (KNAthari) [reply app/views/identifications/index.html.erb:2]: @identifications is defined in the identification controller. I added `_box` to differentiate between the boxes created by the loop.

**Review by crestu — APPROVED** [review]
looks good! got everything to render properly on my end. nice work!

**Review by edithemmings — APPROVED** [review]
Looks great!! Nice work with mapping/sorting that data!! 🎉🚀

**Pull request merged**